gradle.properties

```diff
@@ -56,9 +56,9 @@ easyIDScheme=easyid://
 pomId=yoti-button-sdk
 pomGroup=com.yoti.mobile.android.sdk
 pomPackaging=aar
-pomVersion=1.3.1
+pomVersion=1.3.2
```

Comment:
Hey @pedrojtmartins! My PR with the fix has been just merged into master and published in maven central as 1.3.2. Could you please change the version to 1.3.3? :happy: